lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -570,7 +570,7 @@ if (require.main === module) {
 
   if ((collisions) > 0) {
     console.log(`MEANING COLLISION: ${collisions} string(s) have the same content.`);
-    assert.equal(collisions, 16, `The number of duplicate strings have changed, update this assertion if that is expected, or reword strings. Collisions: ${collisionStrings}`);
+    assert.equal(collisions, 17, `The number of duplicate strings have changed, update this assertion if that is expected, or reword strings. Collisions: ${collisionStrings}`);
```

Comment:
this should only increase when we use an old string in a *new* way. do you know which string was a duplicate? I'm guessing it's `Failing Elements` which wouldn't be used in a new way so we should move it to the i18n class instead :)

Reply:
Ah, I didn't realize the caveat about it needing to be a _new_ way. I'll make sure to move it to the i18n class!